### ISSUE TITLE
Cannot set Reminder on Android

### ISSUE THREAD

**Issue by JmyW** [ja2375/add_2_calendar#137]
Could you please add reminder for Android? The default 30 mins is really not covering all scenario. 
Thanks!

**Comment by dudeck** [comment]
Yes, we need this

**Comment by pgrzywa92** [comment]
Yep, it would be great to have the option to add custom reminders for events.
